Metadata.md

```diff
@@ -1,10 +1,18 @@
 # Additional metadata for SailfishOS:Chum
 
-The basic specification for the additional metadata is that of AppStream.  While it is a goal to leverage AppStream metadata directly for SailfishOS:Chum packages and the SailfishOS:Chum GUI application, we are not at that stage yet, and are instead using a method of embedding metadata in the "Description" field of the rpm spec file.  An example of how this is done follows:
+The basic specification for the additional metadata is that of AppStream.
+While it is a goal to leverage AppStream metadata directly for SailfishOS:Chum packages and the SailfishOS:Chum GUI application, we are not at that stage yet, and are instead using a method of embedding metadata in the `%description` section of the RPM spec file.
+
+Note that it is important, that the part from the `%if "%{?vendor}" == "chum"` to the `%endif` is the last and a contiguous paragraph of the `%description` section, i.e., it must not contain empty or comment lines (or any other line, which is evaluated to an empty line).
```

Comment:
In principle, you don't have to include it within `%if` / structure. Maybe that should be stated as well.

Reply:
Done by [commit `10921ef`](https://github.com/sailfishos-chum/main/pull/70/commits/10921ef18391893df6df241f619ef8c663a4e624).